client/alloc_endpoint.go

```diff
@@ -0,0 +1,76 @@
+package client
```

Comment:
Consider having a sub-folder for RPC endpoints

Reply:
I would be for that maybe down the line during the client refactor. Right now that would mean not having access to the client directly since moving folders would force a different package